lnbits/core/services.py

```diff
@@ -142,24 +142,26 @@ async def pay_invoice(
         if wallet.balance_msat < 0:
             raise PermissionError("Insufficient balance.")
 
-        if internal_checking_id:
-            # mark the invoice from the other side as not pending anymore
-            # so the other side only has access to his new money when we are sure
-            # the payer has enough to deduct from
+    if internal_checking_id:
```

Comment:
Please check the indentation. `internal_checking_id` is set in another scope and will always be `None` here?